CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
 ## 1.0.0 (2024-10-09)
 
 Initial release.
+
+## 2.0.0 (2024-10-10)
```

Comment:
Put the most recent version at the top.

Reply:
Done in latest

Reply:
Did you forgot to push? I don't see any change.

Reply:
All set